icu4c/source/test/Makefile.in

```diff
@@ -86,6 +85,13 @@ all-recursive install-recursive clean-recursive distclean-recursive dist-recursi
 	  $(MAKE) "$$target-local" || exit; \
 	fi
 
+check-recursive: intltest $(IOTEST) cintltst $(LETEST) $(FUZZER)
```

Comment:
Couldn't this use an existing variable such as  `check-recursive: $(SUBDIRS)` ?

Reply:
I am not sure this will work or not. so do not pay attention to this PR (I clearl mark "incomplege") until I feel it is ready for other to read. Otherwise, it is just waste your time.

Reply:
might consider making it draft, incomplete still seems like it's ready for partial review (I was requested to review)